include/SFML/Audio/SoundStream.hpp

```diff
@@ -281,7 +281,8 @@ class SFML_AUDIO_API SoundStream : public SoundSource
 
     enum
     {
-        BufferCount = 3 ///< Number of audio buffers used by the streaming loop
+        BufferCount = 3,    ///< Number of audio buffers used by the streaming loop
+        BufferRetries = 2   ///< Number of retries (excluding initial try) for onGetData()
```

Comment:
> How would that be an issue? You can have duplicate values in the same enum.

guess I (re?)learn something every day, especially when working on SFML! :-)

> The real question is: do this constant need to be exposed in the header?

I assumed it was to be consistent with `BufferCount`, which is apparently only used in the implementation file. Yesterday, I searched the original commit that introduced `BufferCount` to see if a comment explained this decision but the history goes back to the import commit from SVN. Maybe you remember why you put it in the header?

I guess the obvious alternatives are to move this constant either a) to an anonymous namespace in the implementation file, or b) directly in the `fillAndPushBuffer` function. But I somehow like this to be (privately) visible in the class definition as it shows its behaviour -- I wouldn't call this a mere implementation detail, that's why I'd be fine to let it as is (with one enum). Yet, I wouldn't be hurt if the consensus was to move it elsewhere, of course.